apps/src/templates/referenceGuides/ReferenceGuideView.jsx

```diff
@@ -1,21 +1,89 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
+import NavigationBar from './NavigationBar';
+import color from '@cdo/apps/util/color';
+import {Link} from '@dsco_/link';
+import {organizeReferenceGuides} from '@cdo/apps/util/referenceGuideHelpers';
+import classNames from 'classnames';
 
-export default function ReferenceGuideView({referenceGuide}) {
-  return (
-    <div>
-      <h1>{referenceGuide.display_name}</h1>
-      <EnhancedSafeMarkdown markdown={referenceGuide.content} />
-    </div>
-  );
-}
+const baseUrl = window.location.href
+  .split('/')
+  .slice(0, -1)
+  .join('/');
 
 const referenceGuideShape = PropTypes.shape({
   display_name: PropTypes.string,
-  content: PropTypes.string
+  content: PropTypes.string,
+  position: PropTypes.number,
+  parent_reference_guide_key: PropTypes.string
 });
 
+const NavBarItem = ({guide, isActive}) => (
+  <div
+    style={{paddingLeft: `${guide.level * 12}px`}}
+    className={classNames({
+      'nav-link': true,
+      active: isActive
+    })}
+  >
+    <Link className="link" href={`${baseUrl}/${guide.key}`} weight="medium">
+      {guide.display_name}
+    </Link>
+  </div>
+);
+NavBarItem.propTypes = {
+  guide: referenceGuideShape.isRequired,
+  isActive: PropTypes.bool
+};
+
+export default function ReferenceGuideView({referenceGuide, referenceGuides}) {
+  let rootCategory = referenceGuide;
```

Comment:
It seems like this logic to organize the reference guides should go in the nav bar component.